extensions/arc/src/new-window.tsx

```diff
@@ -0,0 +1,11 @@
+import { showHUD } from "@raycast/api";
+import { createNewArcWindow } from "./utils";
+
+export default async function Command() {
+  try {
+    await createNewArcWindow();
+    await showHUD("✅ Window created");
+  } catch {
+    await showHUD("❌ Couldn't create window");
+  }
```

Comment:
Let's not show the HUD in the success case. The newly created window should be enough of a visual feedback.
```suggestion
  try {
    await createNewArcWindow();
  } catch {
    await showHUD("❌ Couldn't create a new window");
  }
```

Reply:
Good point, updated!